src/components/nav/nav-controller.ts

```diff
@@ -1442,54 +1442,69 @@ export class NavController extends Ion {
       // the ElementRef of the actual ion-page created
       let pageElementRef = component.location;
 
-      // a new ComponentRef has been created
-      // set the ComponentRef's instance to its ViewController
-      view.setInstance(component.instance);
+      let promise = null;
+      if ( component.instance['_ionicProjectContent'] ) {
```

Comment:
I tried that.  It loads the content too late in the process to correctly do the navBar stuff.  We need the content to be loaded before the _loadPageCont method kicks off in the PR.